book/src/pong-tutorial/pong-tutorial-03.md

```diff
@@ -140,7 +141,7 @@ impl<'s> System<'s> for PaddleSystem {
 }
 ```
 
-Note: We had to make our Paddle and Side public in `pong.rs`
+Note: We had to make our Paddle, Side, PADDLE_HEIGHT, and ARENA_HEIGHT public in `pong.rs`
```

Comment:
I don't think this line is needed anymore since you already declared those variables as public. What you could do instead is add a note in tutorial 2 that says they are made public for use in other modules later.

Reply:
Good idea, done.